server/src/main/webapp/WEB-INF/incl/layout/jqueryJsonEditor.xhtml

```diff
@@ -14,9 +14,17 @@
 	</h:panelGroup>
 
 	<h:outputScript>  
+		var scopeExpressionJson = "";
+		var scopeExpressionvar = document.getElementById("resourceid:#{id}_hidden").value;
```

Comment:
Is it possible to use jquery?

Reply:
i tried with jquery but not working. so i used core js.